aspnetcore/publishing/iis.md

```diff
@@ -103,21 +103,7 @@ To configure *IISIntegration* service options, include a service configuration f
 
 ### *publish-iis* tool
 
-The *publish-iis* tool can be added to any .NET Core application and will configure the ASP.NET Core Module by creating or modifying the *web.config* file. The tool runs after publishing with the *dotnet publish* command or publishing with Visual Studio and will configure the *processPath* and *arguments* for you. If you're publishing a *web.config* file by including the file in your project and listing the file in the *publishOptions* section of *project.json*, the tool will not modify other IIS settings you have included in the file.
-
-To include the *publish-iis* tool in your application, add entries to the *tools* and *scripts* sections of *project.json*.
-
-<!-- literal_block {"ids": [], "names": [], "highlight_args": {}, "backrefs": [], "dupnames": [], "linenos": false, "classes": [], "xml:space": "preserve", "language": "none"} -->
-
-```none
-
-   "tools": {
-     "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.1.0-preview4-final"
-   },
-   "scripts": {
-     "postpublish": "dotnet publish-iis --publish-folder %publish:OutputPath% --framework %publish:FullTargetFramework%"
-   }
-   ```
+The *publish-iis* tool is included with "Microsoft.NET.Sdk.Web", that is by including `<Project Sdk="Microsoft.NET.Sdk.Web">` in your *.csproj* file. The *publish-iis* tool will configure the ASP.NET Core Module by creating or modifying the *web.config* file. The tool runs after publishing with the *dotnet publish* command or publishing with Visual Studio. Publishing will configure *processPath* and *arguments*. If you're publishing a *web.config* file by including the file in your project, the tool will not modify IIS settings in your *web.config* file.
```

Comment:
"Microsoft.NET.Sdk.Web" -> `Microsoft.NET.Sdk.Web` (code)

> The *publish-iis* tool will configure the ASP.NET Core Module by creating or modifying the *web.config* file.

I guess this was an internal decision there, but AFAICT there is no such thing as a "publish-iis" tool anymore. *web.config* is produced by the `WebConfigTransform` class of the `Microsoft.NETCore.Sdk.Publish.Tasks` assembly. I think that's a bad idea to go this way because someone might actually go looking for info on the "publish-iis" tool ... *and find it when they stumble on a million docs of the pre-2.0 era that no longer apply*, but ok ... if that's what they want to do.

This sentence tho: The tooling doesn't configure the module. The tooling configures *web.config*, which configures the module. Also, the sentence is future tense, if that was something you were trying to avoid. How about? ...

The *publish-iis* tool creates or modifies the *web.config* file, which in turn configures the ASP.NET Core Module.

> Publishing will configure *processPath* and *arguments*.

Publishing configures the *processPath* and *arguments*. (present tense)

> If you're publishing a *web.config* file by including the file in your project, the tool will not modify IIS settings in your *web.config* file.

If you're publishing a *web.config* file by including the file in your project, the tool doesn't modify IIS settings in your *web.config* file.  (present tense)